{"name": "# C5120 Logout from the Sidebar_menu and verify the login page will open after logout", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5120_Logout_from_the_Sidebar_menu.py\", line 8, in clck_usr_nm\n    context.app.main_page.clck_usr_nm()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 357, in clck_usr_nm\n    wait.until(EC.element_to_be_clickable(USR_NM)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1606886491288, "stop": 1606886497168}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1606886497168, "stop": 1606886500066}, {"name": "Then Click on the User name in the Sidebar menu", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5120_Logout_from_the_Sidebar_menu.py\", line 8, in clck_usr_nm\n    context.app.main_page.clck_usr_nm()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 357, in clck_usr_nm\n    wait.until(EC.element_to_be_clickable(USR_NM)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1606886500066, "stop": 1606886523037}, {"name": "Then Hover over the Logout button in the dropdown menu and click on the button Logout", "status": "skipped", "start": 1606886525113, "stop": 1606886525113}, {"name": "And https://devcloud.connectedio.com/ is open", "status": "skipped", "start": 1606886525113, "stop": 1606886525113}], "start": 1606886486139, "stop": 1606886525113, "uuid": "745ecb2d-7b18-4a74-a105-8ba5f3ca8a1a", "historyId": "520fbb859be4ec07e48b2ac30200cc54", "fullName": "features/tests/Login page/C5120_Logout_from_the_Sidebar_menu.feature:# C5120 Logout from the Sidebar_menu and verify the login page will open after logout", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5120 Logout from the Sidebar_menu"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}